packages/calcite-components/src/components/segmented-control/segmented-control.tsx

```diff
@@ -76,19 +72,30 @@ export class SegmentedControl
    */
   @Prop({ reflect: true }) required = false;
 
-  /** Defines the layout of the component. */
-  @Prop({ reflect: true }) layout: Layout = "horizontal";
-
   /**
    * Specifies the name of the component.
    *
    * Required to pass the component's `value` on form submission.
    */
   @Prop({ reflect: true }) name: string;
 
+  /** Specifies the appearance style of the component. */
+  @Prop({ reflect: true }) appearance: Extract<"outline" | "outline-fill" | "solid", Appearance> =
+    "solid";
+
+  /** Defines the layout of the component. */
+  @Prop({ reflect: true }) layout: Layout = "horizontal";
+
   /** Specifies the size of the component. */
   @Prop({ reflect: true }) scale: Scale = "m";
 
+  @Watch("appearance")
+  @Watch("layout")
+  @Watch("scale")
+  handlePropsChange(): void {
```

Comment:
Nitpick: the current name can apply to any sort of prop being modified, so can you rename it to reflect the context or purpose? Maybe `handleItemPropChange`?